src/PrestaShopBundle/Form/Admin/Sell/Product/Pricing/SpecificPriceType.php

```diff
@@ -143,25 +147,13 @@ public function buildForm(FormBuilderInterface $builder, array $options): void
             throw new SpecificPriceException('product_id is required to add/edit specific price.');
         }
 
+        $specificPriceStartDate = new \DateTime('now');
+
         $builder
             ->add('product_id', HiddenType::class)
-            ->add('currency_id', ChoiceType::class, [
-                'label' => $this->trans('Currency', 'Admin.Global'),
-                'placeholder' => $this->trans('All currencies', 'Admin.Global'),
-                'choices' => $this->currencyByIdChoiceProvider->getChoices(),
-                'required' => false,
-            ])
-            ->add('country_id', ChoiceType::class, [
-                'label' => $this->trans('Country', 'Admin.Global'),
-                'placeholder' => $this->trans('All countries', 'Admin.Global'),
-                'choices' => $this->countryByIdChoiceProvider->getChoices(),
-                'required' => false,
-            ])
-            ->add('group_id', ChoiceType::class, [
-                'label' => $this->trans('Group', 'Admin.Global'),
-                'required' => false,
-                'placeholder' => $this->trans('All groups', 'Admin.Global'),
-                'choices' => $this->groupByIdChoiceProvider->getChoices(),
+            ->add('groups', GroupPriceType::class, [
```

Comment:
I'm surprised that this is all it took to match the mockups Only this field, the proper base form theme and the date range were enough?